### PR TITLE
feat: add homepage

### PR DESCRIPTION
**Pull request by tobySolutions**
I was able to add the homepage and redirect users to the registration page as well. Please review this @geromegrignon, @santoshyadavdev; thanks!

This PR is a WIP for #8

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -1,68 +1,50 @@
...
 ---
 
-<Layout title="Welcome to Astro.">
+<Layout title="Home">
 	<main>
+		<div class="hero min-h-screen">
+			<div class="hero-content text-center">
+				<div class="max-w-ld flex flex-col items-center">
+					<h1 class="text-5xl pb-3 font-bold">This Is Learning Conference</h1>
+					<h2 class="text-3xl font-bold">Happening On 16-17 Of May</h2>
```
Comment (geromegrignon) [inline src/pages/index.astro]: I would use the 'Happening on May 16-17' format

File: src/pages/index.astro [hunk 2]
```diff
@@ -1,68 +1,50 @@
...
+					<p class="py-6">Registration Opens On Monday 19th Feb 6am CET</p>
```
Comment (geromegrignon) [inline src/pages/index.astro]: I would remove this part as the registration is already open

File: src/pages/index.astro [hunk 3]
```diff
@@ -1,68 +1,50 @@
...
+					<a href="https://konfhub.com/tc-b34e332d" target="_blank"><button class="btn btn-primary text-2xl font-medium">Register</button></a>
+
+					<div class="stats shadow mt-8 grid-flow-row md:grid-flow-col">
```
Comment (geromegrignon) [inline src/pages/index.astro]: Due to the current state, we should comment this part as we only know the track part (4 tracks).

File: src/components/Footer.astro [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+import {SOCIAL_LINKS} from "../constants";
+---
+<footer class="footer p-10 bg-base-300 text-base-content mt-20">
```
Comment (geromegrignon) [inline src/components/Footer.astro:4]: Need to review with @santoshyadavdev about the links we want to use in the footer

Comment (tobySolutions) [reply src/components/Footer.astro:4]: For sure, @santoshyadavdev needs to review this

**Comment by tobySolutions** [comment]
Wait, @geromegrignon aren't we supposed to deploy somewhere so we can have a staging place where we see our changes in production?

File: src/components/Footer.astro [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  </nav>
+  <nav>
+    <h6 class="footer-title">Company</h6>
+    <a class="link link-hover">About us</a>
+    <a class="link link-hover">Contact</a>
+  </nav>
+  <nav>
+    <h6 class="footer-title">Social</h6>
+    <div class="grid grid-flow-col gap-4">
+      <a href={SOCIAL_LINKS.twitter}><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" class="fill-current"><path d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z"></path></svg></a>
```
Comment (santoshyadavdev) [inline src/components/Footer.astro]: lets use X logo

File: src/components/Footer.astro [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      <a href={SOCIAL_LINKS.youtube}><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" class="fill-current"><path d="M19.615 3.184c-3.604-.246-11.631-.245-15.23 0-3.897.266-4.356 2.62-4.385 8.816.029 6.185.484 8.549 4.385 8.816 3.6.245 11.626.246 15.23 0 3.897-.266 4.356-2.62 4.385-8.816-.029-6.185-.484-8.549-4.385-8.816zm-10.615 12.816v-8l8 3.993-8 4.007z"></path></svg></a>
```
Comment (santoshyadavdev) [inline src/components/Footer.astro:23]: For YT : https://www.youtube.com/@ThisisTechTalks/

File: src/components/Footer.astro [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+      <a href={SOCIAL_LINKS.facebook}><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" class="fill-current"><path d="M9 8h-3v4h3v12h5v-12h3.642l.358-4h-4v-1.667c0-.955.192-1.333 1.115-1.333h2.885v-5h-3.808c-3.596 0-5.192 1.583-5.192 4.615v3.385z"></path></svg></a>
```
Comment (santoshyadavdev) [inline src/components/Footer.astro:24]: we can remove this as of now

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]

**Comment by santoshyadavdev** [comment]
@geromegrignon do you want me to setup netlify deployment?

File: src/components/Footer.astro (hunk 6 above)
Comment (tobySolutions) [reply src/components/Footer.astro:23]: Awesome! Thanks.

File: src/components/Footer.astro (hunk 7 above)
Comment (tobySolutions) [reply src/components/Footer.astro:24]: You are saying i should remove all the logos?

File: src/components/Footer.astro (hunk 6 above)
Comment (tobySolutions) [reply src/components/Footer.astro:23]: Done!

File: src/components/Footer.astro (hunk 5 above)
Comment (tobySolutions) [reply src/components/Footer.astro]: Done!

**Comment by tobySolutions** [comment]
> @geromegrignon do you want me to setup netlify deployment?

@geromegrignon?

**Comment by geromegrignon** [comment]
> > @geromegrignon do you want me to setup netlify deployment?
> 
> @geromegrignon?

It should be handled by Santosh directly from his Netlify account by connecting a site to the repo from there.
There is nothing required on the repo.

It'll create staging deployments for each PR directly.

**Comment by santoshyadavdev** [comment]
Can we merge this? @geromegrignon @tobySolutions

**Comment by tobySolutions** [comment]
> Can we merge this? @geromegrignon @tobySolutions

@geromegrignon's decision is final here.

**Comment by santoshyadavdev** [comment]
The website is deployed here https://main--this-is-learning-conf.netlify.app/

**Comment by tobySolutions** [comment]
> The website is deployed here https://main--this-is-learning-conf.netlify.app/

Awesome!!  Thanks Santosh!

**Comment by santoshyadavdev** [comment]
I did the domain mapping, we need to wait for 24 hrs

**Pull request merged**